core/src/key/namespace/ac.rs

```diff
@@ -1,53 +1,53 @@
-//! Stores a DEFINE TOKEN ON NAMESPACE config definition
+//! Stores a DEFINE ACCESS ON NAMESPACE config definition
 use crate::key::error::KeyCategory;
 use crate::key::key_req::KeyRequirements;
 use derive::Key;
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Key)]
 #[non_exhaustive]
-pub struct Tk<'a> {
+pub struct Ac<'a> {
 	__: u8,
 	_a: u8,
 	pub ns: &'a str,
 	_b: u8,
 	_c: u8,
 	_d: u8,
-	pub tk: &'a str,
+	pub ac: &'a str,
 }
 
-pub fn new<'a>(ns: &'a str, tk: &'a str) -> Tk<'a> {
-	Tk::new(ns, tk)
+pub fn new<'a>(ns: &'a str, ac: &'a str) -> Ac<'a> {
+	Ac::new(ns, ac)
 }
 
 pub fn prefix(ns: &str) -> Vec<u8> {
 	let mut k = super::all::new(ns).encode().unwrap();
-	k.extend_from_slice(&[b'!', b't', b'k', 0x00]);
+	k.extend_from_slice(&[b'!', b'a', b'c', 0x00]);
 	k
 }
 
 pub fn suffix(ns: &str) -> Vec<u8> {
 	let mut k = super::all::new(ns).encode().unwrap();
-	k.extend_from_slice(&[b'!', b't', b'k', 0xff]);
+	k.extend_from_slice(&[b'!', b'a', b'c', 0xff]);
 	k
 }
 
-impl KeyRequirements for Tk<'_> {
+impl KeyRequirements for Ac<'_> {
 	fn key_category(&self) -> KeyCategory {
-		KeyCategory::NamespaceToken
+		KeyCategory::NamespaceAccess
 	}
 }
 
-impl<'a> Tk<'a> {
-	pub fn new(ns: &'a str, tk: &'a str) -> Self {
+impl<'a> Ac<'a> {
+	pub fn new(ns: &'a str, ac: &'a str) -> Self {
 		Self {
 			__: b'/',
 			_a: b'*',
 			ns,
 			_b: b'!',
-			_c: b't',
-			_d: b'k',
-			tk,
+			_c: b'a',
```

Comment:
Also requires migration

Reply:
That is correct. Is that not acceptable for a breaking change in 2.0?

Reply:
Yeah, lets do it. Ideally we would have deprecated before the major, alongside having the preferable option. Then users on say, 1.15 can migrate to this new mechanism and then upgrade to 2.0 where the legacy gets deleted.

For migrating data, I think your export/import is a good solution since it is user facing and query-representable. So this can be marked as resolved.